### DISCUSSION TITLE
Permission Requirements

### DISCUSSION THREAD

**Discussion by alphasierra59** [Arcticons-Team/Arcticons#1394] [Q&A] [answered]
Hi, I was just wondering why this icon pack requires so many more permissions than Frost.

Frost only requires
- set wallpaper
- adjust your wallpaper size

Where Articons requires
- view Wi-Fi connections
- view network connections
- have full network access
- prevent phone from sleeping
- set wallpaper
- android.permission.SET_ALARM
- modify or delete the contents of your shared storage
- read the contents of your shared storage
- run at startup
- run foreground service

I can see why some of these are needed, but why does an icon pack need all of these network permissions or the ability to get to shared storage?

Thanks!

**Comment by Donnnno** [comment]
Good question, we use a modified version of the [Candybar dashboard](https://github.com/zixpo/candybar), it has lots more options than frosts very minimal and outdated one. 

To be honest, I have no idea what all of them do. But good to know, Arcticons doesn't send any data to shady servers. I've even written a tiny privacy policy here: https://github.com/Donnnno/Arcticons/blob/main/Privacy_Policy.md

And I've also removed some in the past, like license checking, for example.

**Reply by Donnnno** [reply, marked as answer]
As described here in the apps FAQ:

![Screenshot_20230213_085609_Arcticons Dark.jpg](https://user-images.githubusercontent.com/31142286/218401628-880353f0-fd54-4ffe-92db-e8f9c9b33e2b.jpg)

**Reply by Donnnno** [reply]
With a more recent update, Arcticons uses **no** internet-related permissions anymore:

- view Wi-Fi connections
- view network connections
- have full network access

**Reply by alphasierra59** [reply]
Thanks for the follow up!
